### DISCUSSION TITLE
Automatic change to the SVC crawler

### DISCUSSION THREAD

**Discussion by ErikBrown2** [TumblThreeApp/TumblThree#370] [Q&A (Question & Answer)]
The SVC crawler will now be preferred by lots of users since it gets also larger images. But changing the setting for each blog can be lots of work if you have a large number of blogs configured. 

The application, after manually making the change for each blog, writes the the changes to the index files while exiting the application. Saving these changes takes the application only a few seconds. So would it not be possible to do the change automatically for all blogs? For instance, there could be a checkbox for this in the settings that, if checked, would trigger all blogs during application exit to be changed automatically to the SVC crawler. Ideally, this checkbox will then be un-checked automatically so that the automatic conversion will not be unnecessarily repeated.

This feature will be very helpful for users that have many blogs listed and are interested in high resolution images.

**Comment by johanneszab** [comment]
To mass change the Crawler of all Tumblr Blogs, do the following:

* Open TumblThree, 
* press "ctrl-A" to select all blogs in the manager, 
* select the _Details_ pane on the right, 
* in the _Crawler_ checkbox, select _Tumblr SVC_. 

This will change the Crawler of Tumblr blogs only (i.e. non-hidden, and hidden Tumblr blogs), but not to Liked-By downloads, or Tumblr Searches.

Note: You can also use the Shift/CTRL key and the mouse to select multiple blogs at once.

**Comment by ErikBrown2** [comment]
Thanks for letting me know. This works indeed great! Unless you did already mention it somewhere and that I missed it, maybe you can mention this on the main page https://github.com/TumblThreeApp/TumblThree since it will be very helpful for the users. I have on several occasions made changes to all the blogs one by one.
